sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/tests/TimeSeriesIdTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.Iot.TimeSeriesInsights.Tests
+{
+    public class TimeSeriesIdTests : E2eTestBase
+    {
+        private static readonly TimeSpan s_retryDelay = TimeSpan.FromSeconds(10);
+
+        // This is the GUID that TSI uses to represent the default type for a Time Series Instance.
+        // TODO: replace hardcoding the Type GUID when the Types resource has been implemented.
+        private const string DefaultType = "1be09af9-f089-4d6b-9f0b-48018b5f7393";
+        private const int MaxNumberOfRetries = 10;
+
+        public TimeSeriesIdTests(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        [Test]
+        public async Task TimeSeriesInsightsInstances_TimeSeries3Id()
```

Comment:
I would rename to something like  TimeSeriesInsightsInstances_Lifecycle_With3Keys in test name

Reply:
Will rename to TimeSeriesId_With3Keys